crates/contract_history/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "contract_history"
+version = "0.1.0"
+edition = "2024"
```

Comment:
question: should we keep editions in sync?

Reply:
Created follow-up issue https://github.com/near/mpc/issues/1035